src/Psalm/CallMap.php

```diff
@@ -7051,108 +7052,96 @@
 'mysqli_report' => ['bool', 'flags'=>'int'],
 'mysqli_result::__construct' => ['void', 'link'=>'mysqli', 'resultmode='=>'int'],
 'mysqli_result::close' => ['void'],
-'mysqli_result::current_field' => ['int'],
 'mysqli_result::data_seek' => ['bool', 'offset'=>'int'],
-'mysqli_result::fetch_all' => ['', 'resulttype='=>'int'],
-'mysqli_result::fetch_array' => ['', 'resulttype='=>'int'],
+'mysqli_result::fetch_all' => ['array', 'resulttype='=>'int'],
+'mysqli_result::fetch_array' => ['array|null', 'resulttype='=>'int'],
 'mysqli_result::fetch_assoc' => ['array<string,string>|null'],
-'mysqli_result::fetch_field' => ['object'],
-'mysqli_result::fetch_field_direct' => ['object', 'fieldnr'=>'int'],
-'mysqli_result::fetch_fields' => ['array'],
-'mysqli_result::fetch_object' => ['object', 'class_name='=>'string', 'params='=>'array'],
-'mysqli_result::fetch_row' => [''],
-'mysqli_result::field_count' => ['int'],
+'mysqli_result::fetch_field' => ['object|false'],
+'mysqli_result::fetch_field_direct' => ['object|false', 'fieldnr'=>'int'],
+'mysqli_result::fetch_fields' => ['array|false'],
+'mysqli_result::fetch_object' => ['object|null', 'class_name='=>'string', 'params='=>'array'],
+'mysqli_result::fetch_row' => ['array|null'],
 'mysqli_result::field_seek' => ['bool', 'fieldnr'=>'int'],
-'mysqli_result::free' => [''],
+'mysqli_result::free' => ['void'],
 'mysqli_result::free_result' => ['void'],
-'mysqli_result::lengths' => ['array'],
-'mysqli_result::num_rows' => ['int'],
-'mysqli_rollback' => ['bool', 'link'=>'mysqli'],
+'mysqli_rollback' => ['bool', 'link'=>'mysqli', 'flags='=>'int', 'name='=>'string'],
 'mysqli_rpl_parse_enabled' => ['int', 'link'=>'mysqli'],
 'mysqli_rpl_probe' => ['bool', 'link'=>'mysqli'],
-'mysqli_rpl_query_type' => ['int', 'query'=>'string', 'link'=>'mysqli'],
+'mysqli_rpl_query_type' => ['int', 'link'=>'mysqli', 'query'=>'string'],
 'mysqli_savepoint' => ['bool', 'link'=>'mysqli', 'name'=>'string'],
 'mysqli_savepoint_libmysql' => ['bool'],
 'mysqli_select_db' => ['bool', 'link'=>'mysqli', 'dbname'=>'string'],
-'mysqli_send_query' => ['bool', 'query'=>'string', 'link'=>'mysqli'],
-'mysqli_set_charset' => ['bool', 'link'=>'mysqli', 'csname'=>'string'],
-'mysqli_set_local_infile_default' => ['', 'link'=>'mysqli'],
+'mysqli_send_query' => ['bool', 'link'=>'mysqli', 'query'=>'string'],
+'mysqli_set_charset' => ['bool', 'link'=>'mysqli', 'charset'=>'string'],
+'mysqli_set_local_infile_default' => ['void', 'link'=>'mysqli'],
 'mysqli_set_local_infile_handler' => ['bool', 'link'=>'mysqli', 'read_func'=>'callable'],
 'mysqli_slave_query' => ['bool', 'link'=>'mysqli', 'query'=>'string'],
 'mysqli_sqlstate' => ['string', 'link'=>'mysqli'],
 'mysqli_ssl_set' => ['bool', 'link'=>'mysqli', 'key'=>'string', 'cert'=>'string', 'ca'=>'string', 'capath'=>'string', 'cipher'=>'string'],
-'mysqli_stat' => ['', 'link'=>'mysqli'],
-'mysqli_stmt::__construct' => ['void', 'link='=>'mysqli', 'query='=>'string'],
-'mysqli_stmt::affected_rows' => ['int'],
```

Comment:
I wanted to point out that I did remove a few lines like this one where they're not actual methods, but rather properties on the class (http://php.net/manual/en/mysqli-stmt.affected-rows.php).  I assume this is correct (as I understand it, this file is for functions/methods, not class properties). If I'm wrong in that assumption, let me know and I can re-instate.

I did confirm that the ones I removed aren't actually functions via reflection:

```
$ref = new ReflectionMethod('mysqli_stmt', 'affected_rows');
PHP Warning:  Uncaught ReflectionException: Method mysqli_stmt::affected_rows() does not exist in php shell code:1
Stack trace:
#0 php shell code(1): ReflectionMethod->__construct('mysqli_stmt', 'affected_rows')
#1 {main}
  thrown in php shell code on line 1
```